tests/integration/utils.py

```diff
@@ -126,7 +126,7 @@ def fetch_content_manifest(self, request_id):
         """
         resp = requests.get(f"{self._cachito_api_url}/requests/{request_id}/content-manifest")
         resp.raise_for_status()
-        return Response(resp.json(), resp.json()["id"], resp.status_code)
+        return Response(resp.json(), request_id, resp.status_code)
```

Comment:
In the future, this would be better as a separate commit as it's not related to the new test being added but you can keep it as is for this PR.

Reply:
@mprahl Thank you! I just separated it for two commits.  :)